src/menus/menu.jsx

```diff
@@ -206,7 +206,7 @@ const Menu = React.createClass({
     rootStyle.transition = Transitions.easeOut('250ms', ['opacity', 'transform']);
     rootStyle.transform = 'translate3d(0,-8px,0)';
     rootStyle.opacity = 0;
-    rootStyle = AutoPrefix.all(rootStyle);
+    rootStyle = autoPrefix.all(rootStyle, this.state.muiTheme);
```

Comment:
`set` might be a bit hard to remove. but what aout `all`? I feel like it's the same as calling `theme.prefix`.